st/clitests/awss3api_spec.py

```diff
@@ -369,24 +369,74 @@ def delete_object_list_file(file_name):
 delete_object_list_file(hierarchical_object_list_file)
 
 
-# ************ CopyObject API supported *************************
+# ************ CopyObject API supported *****************************************************************
+
 AwsTest('Aws can put object').put_object("seagatebucket", "1kfile", 1024)\
     .execute_test().command_is_successful()
 
-AwsTest('Aws can copy object').copy_object("seagatebucket/1kfile", "seagatebucket", "1kfile-copy")\
+# Positive: copy-object from a bucket to the same bucket as destination.
+AwsTest('Aws can copy object to same bucket').copy_object("seagatebucket/1kfile", "seagatebucket", "1kfile-copy")\
```

Comment:
you can add test to copy file of 0 bytes

Reply:
I have done the manual testing of this test. As discussed, not adding this test to system test file.